client/edge_connectivity_test.go

```diff
@@ -0,0 +1,413 @@
+package client
```

Comment:
Per the discussion, i thought this was moving over to test/integration

Reply:
I talked to Nicolas about that and he ended up deciding that it should stay where it is.
I'm afraid I don't recall the reasoning now...